### PR TITLE
Prepare 2.14.0 release

### PR DESCRIPTION
**Pull request by bessbd**
# 2.14.0 (2020-08-17)

- [FIXED] Set default value for `partitioned` parameter to false when creating a design document.
- [FIXED] Corrected setting of `partitioned` flag for `create_query_index` requests.
- [FIXED] Added a workaround for installation on Python 2.

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # UNRELEASED
```
Comment (emlaver) [inline CHANGES.md:1]: Let's remove this.  You can add it in when creating the PR to update the version to snapshot in Phase III: https://github.ibm.com/cloudant/integrations/blob/master/releasing/README.md#update-the-version-number

Comment (ricellis) [reply CHANGES.md:1]: Yeah this line needs to go. I think it will break the automated release stuff.

Comment (bessbd) [reply CHANGES.md]: Done in 3f4ae5d7c3dc79a5b85a2b481b998570c262b0fd

**Comment by bessbd** [comment]
Thank you for the review, @emlaver and @ricellis . I'm about to merge this.

**Pull request merged**